Update continuous-verification-faqs.md (#7270)
harness/developer-hub: docs/continuous-delivery/verify/continuous-verification-faqs.md

Can you check if Continuous Verification is selected for the configured monitored service health source under **Assign**.
You can select the services for which you want to apply the metric.

### Can we add multiple node filters while using Prometheus?

Yes, you can use a multiple-node filter. All of those filters should be applied.

> ### How to select the correct Service Instance Identifier (SII)

Reviewer: ```suggestion
### How do you select the correct Service Instance Identifier (SII)?
```

> It depends upon what kind of metric you want to monitor. For K8S deployments, the best SII selection is typically pod, podname, or containername.

Reviewer: ```suggestion
It depends upon what kind of metric you want to monitor. For K8s deployments, the best SII selection is typically pod, podname, or containername.
```

Can you check if Continuous Verification is selected for the configured monitored service health source under **Assign**.
You can select the services for which you want to apply the metric.

### Can we add multiple node filters while using Prometheus?

Yes, you can use a multiple-node filter. All of those filters should be applied.

> 
> 

Reviewer: ```suggestion
```

> ### How to select the correct Service Instance Identifier (SII )

Reviewer: ```suggestion
### How to select the correct Service Instance Identifier (SII)
```
